pkg/master/master.go

```diff
@@ -242,6 +243,10 @@ func (m *Master) InstallAPIs(c *Config) {
 			Scheme:                 api.Scheme,
 			ParameterCodec:         api.ParameterCodec,
 			NegotiatedSerializer:   api.Codecs,
+			SubresourceGroupVersionKind: map[string]unversioned.GroupVersionKind{
```

Comment:
If you do need to turn something off when autoscaling isn't on, should omit this, too.